js/app/controllers/content-product.ng

```diff
@@ -218,72 +227,79 @@ $table | Remove-BqTable -Force
     </p>
     <pre>
 # Creates a new table with the Schema object from above.
-$table = $dataset | New-BqTable "my_table" -Schema $schema
+$table = $dataset | New-BqTable "logs2014" -Schema $schema
     </pre>
     
     <h2>TableRow</h2>
     <p>
-      Data is added and removed from Tables in Rows. These Rows are accessible using the 
+      Data is added and removed from <code>Tables</code> in <em>Rows</em>. These rows are accessible using the 
       <cmdlet-ref name="Add-BqTableRow"></cmdlet-ref> and <cmdlet-ref name="Get-BqTableRow"></cmdlet-ref>
-      cmdlets. Add-BqTableRow takes CSV, JSON, and AVRO files to import into BigQuery.
+      cmdlets. <code>Add-BqTableRow</code> takes CSV, JSON, and AVRO files to import into BigQuery.
     </p>
     <pre>
 # Ingests a CSV file and appends its rows onto the table 'my_dataset:my_table'.
-$table = New-BqTable "my_table" -DatasetId "my_dataset"
+$table = New-BqTable "logs2014" -DatasetId "page_views"
 $table | Add-BqTableRow CSV $filename -SkipLeadingRows 1 -WriteMode WriteAppend
 
-# Returns a list of the rows in 'book_data:classics'.
-$list = Get-BqTable "classics" -DatasetID "book_data" | Get-BqTableRow
+# Returns a list of the rows in 'page_views:logs2014'.
+$list = Get-BqTable "logs2014" -DatasetID "page_views" | Get-BqTableRow
     </pre>
     
     <h2>Jobs</h2>
     <p>
-      There are four types of jobs: Query, Load, Extract, and Copy. Query jobs run SQL style queries and 
-      output results to tables. Load jobs import Google Cloud Storage files into BigQuery. Extract jobs 
-      export BigQuery tables to GCS. Copy jobs copy an existing table to another new or existing table.
-      <cmdlet-ref name="Start-BqJob"></cmdlet-ref> starts any of these kinds of jobs as an asynchronous 
+      There are four types of <code>Jobs</code>: Query, Load, Extract, and Copy. <em>Query</em> jobs run SQL style 
+      queries and output results to tables. <em>Load</em> jobs import Google Cloud Storage files into BigQuery. 
+      <em>Extract</em> jobs export BigQuery tables to GCS. <em>Copy</em> jobs copy an existing table to another new or 
+      existing table. <cmdlet-ref name="Start-BqJob"></cmdlet-ref> starts any of these kinds of jobs as an asynchronous 
       operation. Use the <code>-PollUntilComplete</code> flag to have the cmdlet block until the job is done.
       <cmdlet-ref name="Receive-BqJob"></cmdlet-ref> will return the results of a query job once it is 
       finished. <cmdlet-ref name="Get-BqJob"></cmdlet-ref> will return a reference object detailing the
       current state and statistics on the job.  <cmdlet-ref name="Stop-BqJob"></cmdlet-ref> will send a 
       request to the server to stop a certain job, and then returns immediately.
+      <blockquote>
+      Note on formatting table names within query strings: BigQuery format specifies that table names 
+      should be surrounded by backticks (`), but this is Powershell's "continue on the next line" 
```

Comment:
The problem is not that it is the "continue on the next line" operator, the problem is it is PowerShell's escape character.